plugins/kubernetes/src/components/K8sTerminal/K8sTerminalDialog.tsx

```diff
@@ -49,7 +49,7 @@ const useStyles = makeStyles((theme: Theme) =>
  */
 export const K8sTerminalDialog = (props: K8sTerminalProps) => {
   const classes = useStyles();
-  const { containerStatus, podScope } = props;
+  const { clusterName, containerName, podName } = props;
```

Comment:
Nit, destructuring is normally done in arguments rather that after for functional components

Reply:
If we discourage destructuring in public facing components, might as well do it inside the body all the time for consistency, that's why I have it inside.